src/components/breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import { Meta, StoryObj } from "@storybook/react";
+import { BreadCrumbs } from "./Breadcrumbs";
+
+const meta = {
+  title: "Old/BreadCrumbs",
```

Comment:
question(non-blocking): what's the old parameter?

Reply:
The title expresses the path in Storybook to the component. `Old/` creates a directory called "Old" - I've chosen this to differentiate it from newly built components as part of building the component library. This will likely change / evolve / get removed as work continues.